src/lighthouse.php

```diff
@@ -85,6 +85,7 @@
     'cache' => [
         'enable' => env('LIGHTHOUSE_CACHE_ENABLE', env('APP_ENV') !== 'local'),
         'key' => env('LIGHTHOUSE_CACHE_KEY', 'lighthouse-schema'),
+        'store' => env('LIGHTHOUSE_CACHE_STORE', null),
```

Comment:
Can you add a small docblock to each option?

This one might be:

```suggestion

        /*
         * Allows using a specific cache store, uses the app's default if set to null.
         */
        'store' => env('LIGHTHOUSE_CACHE_STORE', null),
```